e2e/cypress/utils/file.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// Converts a file size in bytes into a human-readable string of the form '123MB'.
+export function fileSizeToString(bytes) {
+    // it's unlikely that we'll have files bigger than this
+    if (bytes > 1024 ** 4) {
+        // check if file is smaller than 10 to display fractions
```

Comment:
Quick question: are fractions not displayed if file is greater than 10?

Reply:
This is an exact copy of https://github.com/mattermost/mattermost-webapp/blob/master/utils/utils.jsx#L1230-L1253 which I missed to mention or indicate somewhere here. So, it will have the same behavior as the webapp itself.